src/functionalTest/java/uk/gov/hmcts/reform/blobrouter/CrimeBlobDispatchingTest.java

```diff
@@ -0,0 +1,64 @@
+package uk.gov.hmcts.reform.blobrouter;
+
+import com.azure.storage.blob.BlobServiceClient;
+import com.azure.storage.blob.BlobServiceClientBuilder;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Disabled;
+import org.junit.jupiter.api.Test;
+
+import java.util.concurrent.TimeUnit;
+
+import static com.google.common.io.Resources.getResource;
+import static com.google.common.io.Resources.toByteArray;
+import static com.jayway.awaitility.Awaitility.await;
+import static java.util.Arrays.asList;
+import static uk.gov.hmcts.reform.blobrouter.envelope.ZipFileHelper.createZipArchive;
+import static uk.gov.hmcts.reform.blobrouter.storage.StorageHelper.blobExists;
+import static uk.gov.hmcts.reform.blobrouter.storage.StorageHelper.uploadFile;
+
+public class CrimeBlobDispatchingTest extends FunctionalTestBase {
+
+    private BlobServiceClient crimeStorageClient;
+
+    @Override
+    @BeforeEach
+    protected void setUp() {
+        super.setUp();
+
+        this.crimeStorageClient = new BlobServiceClientBuilder()
+            .connectionString(config.crimeStorageConnectionString)
+            .buildClient();
+    }
+
+    @Disabled("Functionality not implemented, yet")
```

Comment:
it can now be enabled

Reply:
crime not yet implemented actually

Reply:
ok. I assume functional test for cft follows shortly after then? @lgonczar ?

Reply:
This is strictly for crime. It will be possible to reuse a lot of code for CFT,  but there is more work there - SAS token retrieval. This is strictly for crime blobs.